src/asynk/async_queue.rs

```diff
@@ -338,9 +359,26 @@ where
 
     async fn remove_task_query(
         transaction: &mut Transaction<'_>,
-        task: Task,
+        id: Uuid,
+    ) -> Result<u64, AsyncQueueError> {
+        Self::execute_query(transaction, REMOVE_TASK_QUERY, &[&id], Some(1)).await
+    }
+
+    async fn remove_task_by_uniq_hash_query(
```

Comment:
maybe it can be called remove_tasks_by_metadata. so it removes tasks with the same metadata

Reply:
[Rename to remove_task_by_metadata](https://github.com/ayrat555/fang/pull/90/commits/39f5f94ab80a8dd4645e7d39636d1573799938ee)